README.md

```diff
@@ -117,15 +117,15 @@ $ make
 
 ### Build for Wayland backend (weston desktop-shell)
 
+This binary will run as a desktop-shell by setting `weston.ini` when Weston starts. See [Settings of weston.ini file](#4-settings-of-westonini-file-only-when-you-use-weston-desktop-shell).
```

Comment:
The reference is incorrect.  
`(#4-settings-of-westonini-file-only-when-you-use-weston-desktop-shell)`  ->  `(#5-...)`

Reply:
I fixed the link.